### PR TITLE
fix: Fix audit ci 

### PR DESCRIPTION
**Pull request by tommasini**
## **Description**

Since the issue on IP on audit ci it's only for development tools, we ignore it for now to enable ci.

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: .iyarc [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # ReDoS vulnerability, no impact to this application, and fix not backported yet to the versions we use
+
 GHSA-c2qf-rxjj-qqgw
+GHSA-2p57-rm9w-gvfp
```
Comment (Gudahtt) [inline .iyarc:4]: Could you add a comment to explain this? The above comment does not apply here (this is not a ReDoS vulnerability)

Comment (tommasini) [reply .iyarc:7]: Yes! Thanks Mark! 
Added here: https://github.com/MetaMask/metamask-mobile/pull/9831/commits/35199db1248f03802e91922cf61676db9b38264b

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Review by Cal-L — APPROVED** [review]
LGTM

**Review by NicolasMassart — APPROVED** [review]

**Review by NEllusion — APPROVED** [review]

**Pull request merged**